docs/books/modules/user-guide/network-topologies.adoc

```diff
@@ -62,11 +61,22 @@ of the configuration:
   `edge` role.  An edge router may be connected to more than one interior
   router but only one of the edge connections shall be active at one time.
   Edge routers may not establish connections to other edge routers.  They are
-  pure leaves in the overall network topology.  Because there are no limits to
-  the number of edge routers deployed, they may be proliferated in high
+  pure leaves in the overall network topology.  They may be proliferated in high
   numbers as sidecars in a container environment or one-per-host or
   one-per-virtual-machine in a data center.
 
+[NOTE]
+====
+Performance of your router network is determined by various factors:
+
+* topology
+* number of routers
+* underlying infrastructure (host resources, network speed, etc)
+
+It is recommended to implement an architecture that minimises the number 
```

Comment:
I'm a little uncomfortable with this recommendation - it can be interpreted as discouraging redundancy, which may be a key benefit to a particular use case.

The topic of router "performance" is a complex one - I don't think a simple note like this adds much value.  Probably better to simply remove this note.

Reply:
done